Matplotlib/solution/01 - Getting started with Pyplot.ipynb

```diff
@@ -0,0 +1,974 @@
+{
```

Comment:
I believe this 'solution' folder is mistakenly created. Can you please remove this folder @evanyeohboonkhai ? I will make the merge soon after. The rest looks good to me.